src/storage-types/src/sources.rs

```diff
@@ -19,6 +19,8 @@ use std::time::Duration;
 
 use arrow::array::{Array, ArrayRef, BinaryArray, BinaryBuilder, NullArray, StructArray};
 use arrow::datatypes::{Field, Fields};
+#[allow(unused_imports)]
+use base64::Engine;
```

Comment:
Move the import to `mod tests`, around line 2004. Then you won't need to allow unused.